client/pages/search-results.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Result from '../components/result';
+import Map from '../components/map';
+
+export default class SearchResults extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      results: [],
+      center: null
+    };
+  }
+
+  componentDidMount() {
+    const url = new URL('localhost:3000/search-results');
```

Comment:
We want to avoid hard-coding the current url's host because we need to deploy the app to another host on the public internet later. 

```js
const url = new URL('/search-results', window.location);
```

https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#syntax